test/e2e/tests/tokens/add-hide-token.spec.js

```diff
@@ -155,9 +158,13 @@ describe('Add token using wallet_watchAsset', function () {
           .withPermissionControllerConnectedToTestDapp()
           .build(),
         ganacheOptions: defaultGanacheOptions,
+        smartContract,
```

Comment:
in this test, we are adding a token with a hardocoded address, but that address is not a contract (doesn't exist in ganache, it's a clean network, so that's just an empty EOA). With this change, we first deploy an ERC20 token and then use that address for adding it using the wallet_watchAsset method